res/controllers/EKS-Otus.js

```diff
@@ -729,9 +729,13 @@ EksOtus.activateSpinningPlatterLEDs = function() {
 EksOtus.enableSpinningPlatterLEDs = function() {
     if (EksOtus.activeSpinningPlatterGroup==undefined)
         return;
+    // TODO: This control is updated on each audio callback which is
+    // way more often than what's necessary to update LEDs. It would
+    // make more sense (and save CPU) to hook up to [App],indicator_250ms
+    // instead and read the playposition on each callback.
```

Comment:
Maybe we can drop that comment.
Currently, `playposition` is updated 15 times persecond, ie. every 0.0666 ms

Reply:
If I look at this video: https://youtu.be/LO7nWuw8PuQ?t=20
the update rate must be even higher here. For 33rpm and 64 LEDs per round we've a time difference of 28ms between two position LEDs.

Reply:
Okay, true. So [`kPlaypositionUpdateRate`](https://github.com/mixxxdj/mixxx/blob/476123ba5290e2c359fa313a1efa36204d36ea9d/src/engine/enginebuffer.cpp#L51) had to be increased, or a CO being added to customize the update rate as required.

FWIW we also have this comment in the script
https://github.com/mixxxdj/mixxx/blob/81b9e38f8a0e4756721a8838a2d12a10aa9b208a/res/controllers/EKS-Otus.js#L764-L766

I'll remove the comments.

Reply:
Nice, but doesn't matter here since we currently don't have a high-res position control anyway.

Please remember this is only about making this script work again. We can also remove the playpos connection altogether and add a TODO (+ log a warning) to the script and to the manual. I don't care.

Reply:
Yes, please just remove these unrelated ToDos und reduce this PR to the bugfixes itself.